### PR TITLE
Automatic cleanup!

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,25 +21,25 @@
...
   "homepage": "https://github.com/PolymerElements/paper-button",
   "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0-rc.1",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#2.0-preview",
-    "paper-behaviors": "PolymerElements/paper-behaviors#2.0-preview",
-    "paper-styles": "PolymerElements/paper-styles#2.0-preview"
+    "polymer": "Polymer/polymer#1.9 - 2",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#1 - 2",
+    "paper-behaviors": "PolymerElements/paper-behaviors#1 - 2",
+    "paper-styles": "PolymerElements/paper-styles#1 - 2"
```
Comment (keanulee) [inline bower.json]: This isn't a problem for this repo, but does the bot support cases where an element has a dependency that's greater than 1.0.0 (e.g. `1.3.1 - 2`)?

Comment (rictic) [inline bower.json]: It's not super practical for it to look at the version range of the repo at master. I think in practice we'll be saved by the way that bower prefers more recent versions. We'd only run into trouble if someone had given a really low upper bound to a dependency, and the failure mode is that they wouldn't be alerted to the conflict.

It's possible to take this into account in tedium, but it'd be many hours of work

Comment (keanulee) [inline bower.json]: It's probably fine - just wanted to make sure we're aware of it.

File: bower.json [hunk 2]
```diff
@@ -21,25 +21,25 @@
...
-    "iron-icons": "PolymerElements/iron-icons#2.0-preview",
-    "iron-test-helpers": "PolymerElements/iron-test-helpers#2.0-preview",
+    "iron-component-page": "PolymerElements/iron-component-page#1 - 2",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#1 - 2",
+    "iron-icon": "PolymerElements/iron-icon#1 - 2",
+    "iron-icons": "PolymerElements/iron-icons#1 - 2",
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#1 - 2",
     "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
     "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.6",
     "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.1"
```
Comment (notwaldorf) [inline bower.json]: We will also need to update wct and webcomponentsjs

File: bower.json [hunk 3]
```diff
@@ -21,25 +21,25 @@
...
   },
   "variants": {
     "1.x": {
       "dependencies": {
-        "polymer": "Polymer/polymer#^1.0.0",
+        "polymer": "Polymer/polymer#^1.9",
```
Comment (notwaldorf) [inline bower.json]: should this be 1.9.0?

File: bower.json (hunk 1 above)
Comment (stramel) [inline bower.json]: @rictic @keanulee  Just curious if there is a benefit to `1.9 - 2` vs `^1.9 || ^2.0`

Comment (keanulee) [inline bower.json]: Semantically they mean the same thing, so it's just a style choice.

Comment (stramel) [inline bower.json]: Thanks @keanulee! I was pretty sure just wanted to make sure!

**Comment by keanulee** [comment]
Closing since `master` has already been updated with similar changes.

**Pull request closed without merging**